src/uw_basic.jl

```diff
@@ -123,7 +123,15 @@ $(SIGNATURES)
 Compute resonance frequency of a freely oscillating has bubble in water, given:
 - bubble `radius` in meters
 - `depth` of bubble in water in meters
+- γ : gas ratio of specific heats. Default 1.4 for air 
+- p0: atmospheric pressure. Default 1.013e5 Pa
+- ρ_water: Density of water. Default 1022.476 kg/m³
 
+This ignores surface-tension, thermal, viscous and acoustic damping effects, and the pressure-volume relationship is taken to be adiabatic.
 Implementation based on Medwin & Clay (1998).
 """
-bubbleresonance(radius, depth=0.0) = 3.25/radius * √(1+0.1*depth)
+function bubbleresonance(radius, depth=0.0, γ = 1.4, p0 = 1.013e5, ρ_water = 1022.476)
```

Comment:
Prefer not using variable names with underscore unless absolutely necessary, as recommended in the Julia style guide. The rest of the code uses short variables with suffixes. Shall we use just `ρ` instead, since there is no other density involved?

And while you're at it, avoid spaces around `=` in default args in function definition for style consistency with rest of the code please.